pyquil/operator_estimation.py

```diff
@@ -814,11 +816,101 @@ def serializable(self):
         }
 
 
+def calibrate_observable_estimates(qc: QuantumComputer, expt_results: List[ExperimentResult],
+                                   n_shots: int = 500, symm_type: int = -1,
+                                   noisy_program: Optional[Program] = None,
+                                   active_reset: bool = False,
+                                   show_progress_bar: bool = False) \
+        -> Iterable[ExperimentResult]:
+    """
+    Calibrates the expectation and std_err of the input expt_results and updates those estimates.
```

Comment:
```suggestion
    Calibrates the expectation and std_err of the input ``expt_results`` and updates those estimates.
```

I can't remember if it's one backtick or two, but whatever tells sphinx to format as code. Ditto other variables mentioned in this docstring.